### PR TITLE
aws: added new resources

### PR DESCRIPTION
**Pull request by marcoldp**
#299

### CODE REVIEW

File: aws/resources.go [hunk 1]
```diff
@@ -1328,6 +1342,145 @@ func ec2TransitGatewayMulticastDomain(ctx context.Context, a *aws, resourceType
...
+	resources := make([]provider.Resource, 0)
+	for _, tableID := range routeTablesIDs {
+		input := &ec2.GetTransitGatewayRouteTableAssociationsInput{
+			TransitGatewayRouteTableId: awsSDK.String(tableID),
+		}
+		associations, err := a.awsr.GetTransitGatewayRouteTableAssociations(ctx, input)
+		if err != nil {
+			return nil, err
+		}
+		// to test with resourceID
```
Comment (xescugc) [inline aws/resources.go:1443]: ?

File: aws/resources.go [hunk 2]
```diff
@@ -1328,6 +1342,145 @@ func ec2TransitGatewayMulticastDomain(ctx context.Context, a *aws, resourceType
...
+	resources := make([]provider.Resource, 0)
+	for _, tableID := range routeTablesIDs {
+		input := &ec2.GetTransitGatewayRouteTablePropagationsInput{
+			TransitGatewayRouteTableId: awsSDK.String(tableID),
+		}
+		propagations, err := a.awsr.GetTransitGatewayRouteTablePropagations(ctx, input)
+		if err != nil {
+			return nil, err
+		}
+		// to test with resourceID
```
Comment (xescugc) [inline aws/resources.go:1471]: ?

File: aws/resources.go (hunk 1 above)
Comment (marcoldp) [reply aws/resources.go:1443]: was a comment that i forgot to remove, doing it now

File: aws/resources.go (hunk 2 above)
Comment (marcoldp) [reply aws/resources.go:1471]: was a comment that i forgot to remove, doing it now

**Review by talset — APPROVED** [review]

**Review by xescugc — APPROVED** [review]

**Pull request merged**